### PR TITLE
Downshift v5

### PR DESCRIPTION
**Pull request by mshwery**
This upgrades to downshift v5.

- [x] `yarn upgrade downshift --latest`
- [x] removed deprecated props
- [x] fixed touch handler
- [x] `npx yarn-deduplicate && yarn install`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -210,7 +210,7 @@
   "size-limit": [
     {
       "path": "commonjs/index.js",
-      "limit": "240 KB"
+      "limit": "241 KB"
```
Comment (akleiner2) [inline package.json:213]: 😬

File: src/autocomplete/src/Autocomplete.js [hunk 2]
```diff
@@ -199,8 +174,13 @@ export default class Autocomplete extends PureComponent {
 
     if (items.length === 0) return null
 
+    // Pass the actual DOM ref to downshift, this fixes touch support
+    const menuProps = getMenuProps()
```
Comment (akleiner2) [inline src/autocomplete/src/Autocomplete.js:178]: looks like some of these changes are in #793 too?

File: src/autocomplete/src/Autocomplete.js [hunk 3]
```diff
@@ -50,22 +49,6 @@ export default class Autocomplete extends PureComponent {
      */
     selectedItem: PropTypes.any,
 
-    /**
-     * The selected item to be selected & shown by default on the autocomplete (deprecated)
-     */
-    defaultSelectedItem: deprecated(
```
[the remark this answers is not in the document]
Comment (mshwery) [reply src/autocomplete/src/Autocomplete.js:56]: It is – hence targeting `v5` branch and will release as an RC of evergreen-ui@v5

File: src/autocomplete/src/Autocomplete.js (hunk 2 above)
Comment (mshwery) [reply src/autocomplete/src/Autocomplete.js:178]: Yep I realized we can ship these sooner, so I will remove them from this one.

File: src/autocomplete/src/Autocomplete.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (akleiner2) [reply src/autocomplete/src/Autocomplete.js:56]: Ahh i missed the `v5` target branch :). nice!

File: src/autocomplete/src/Autocomplete.js (hunk 2 above)
Comment (akleiner2) [reply src/autocomplete/src/Autocomplete.js:178]: 👍

**Comment by mshwery** [comment]
Published as `5.0.0-alpha-0` in `evergreen-ui@next`
```sh
yarn release 5.0.0-alpha-0 --tag=next --any-branch --no-release-draft
```

**Pull request merged**